jsapp/js/components/modalForms/connectProjects.es6

```diff
@@ -0,0 +1,367 @@
+      return (
+        <ToggleSwitch
+          onChange={this.toggleSharingData.bind(this)}
+          label={t('Data sharing enabled')}
+          checked={this.state.isShared}
+        />
+      );
+    } else {
+      return (
+        <ToggleSwitch
+          onChange={this.toggleSharingData.bind(this)}
+          label={t('Data sharing disabled')}
+          checked={this.state.isShared}
+        />
+      );
+    }
+  }
+  renderSelect(sharingEnabledAssets) {
+    const selectClassNames = ['kobo-select__wrapper'];
+    if (this.state.fieldsErrors.emptyParent || this.state.fieldsErrors.parent) {
+      selectClassNames.push('kobo-select__wrapper--error');
+    }
+    return(
+      <div className={selectClassNames.join(' ')}>
+        <Select
+          placeholder={t('Select a different project to import data from')}
+          options={sharingEnabledAssets}
+          value={this.state.newParent}
+          isLoading={(this.state.isVirgin || this.state.isLoading || !sharingEnabledAssets)}
+          getOptionLabel={option => option.name}
+          getOptionValue={option => option.url}
+          noOptionsMessage={() => {return t('No projects to connect')}}
+          onChange={this.onParentChange}
+          className='kobo-select'
+          classNamePrefix='kobo-select'
+        />
+        <label className='select-errors'>
+          {this.state.fieldsErrors.emptyParent || this.state.fieldsErrors.parent}
+        </label>
+      </div>
+    );
+  }
+
+  render() {
+    let sharingEnabledAssets = [];
+    if (this.state.sharingEnabledAssets !== null) {
+      sharingEnabledAssets = this.generateFilteredAssetList();
+    }
+
+    return (
+      <bem.FormModal__form
+        className='project-settings project-settings--upload-file connect-projects'
+        onSubmit={this.confirmAttachment}
+      >
+        {/* Enable data sharing */}
+        <bem.FormModal__item m='data-sharing'>
+          <div className='connect-projects-header'>
+            <i className="k-icon k-icon-folder-out"/>
+            <h2>{t('Share data with other project forms')}</h2>
+          </div>
+          <p>
+            {t('Enable data sharing to allow other forms to import and use dynamic data from this project. Learn more about dynamic data attachments')}
+            <a href='#'>{t(' ' + 'here')}</a>
+          </p>
+          {this.renderSwitchLabel()}
+        </bem.FormModal__item>
+
+        {/* Attach other projects data */}
+        <bem.FormModal__item m='import-data'>
+          <div className='connect-projects-header'>
+            <i className="k-icon k-icon-folder-in"/>
+            <h2>{t('Import other project data')}</h2>
+          </div>
+          <p>
+            {t('Connect with other project(s) to import dynamic data from them into this project. Learn more about dynamic data attachments')}
+            <a href='#'>{t(' ' + 'here')}</a>
+          </p>
+          {/* Selecting project form*/}
+          {sharingEnabledAssets &&
```

Comment:
**renderSelect 2/3** 

Instead of checking it here, move whole `<div className='import-data-form'>` into `renderSelect`, and here leave only `{this.renderSelect()}`